config/content.json

```diff
@@ -5,8 +5,22 @@
             "/docs/": "https://github.com/rackerlabs/docs-quickstart/",
             "/docs/user-guides/infrastructure/": "https://github.com/rackerlabs/docs-core-infra-user-guide/",
             "/docs/rack-cli/": "https://github.com/rackspace/rack/",
-            "/docs/cloud-images/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-images",
-            "/docs/cloud-load-balancers/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-load-balancers"
+            "/docs/cloud-images/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-images/",
+            "/docs/cloud-load-balancers/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-load-balancers/",
+        "/docs/cloud-block-storage/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-block-storage/",
+        "/docs/cloud-dns/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-dns/",
+        "/docs/cloud-cdn/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-cdn/",
+        "/docs/cloud-databases/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-databases/",
+        "/docs/cloud-backup/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-backup/v1/",
+        "/docs/cloud-orchestration/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-orchestration/",
+        "/docs/rackconnect/v3/developer-guide/": "https://github.com/rackerlabs/docs-cloud-rackconnect/",
+        "/docs/cloud-queues/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-queues/",
+        "/docs/cloud-networks/v1/developer-guide/": "https://github.com/rackerlabs/docs-cloud-networks/",
+        "/docs/cloud-big-data/v2/developer-guide/": "https://github.com/rackerlabs/docs-cloud-big-data/v2/",
```

Comment:
This doesn't match the [content ID for Cloud Big Data](https://github.com/rackerlabs/docs-cloud-big-data/blob/master/rst/dev-guide/cloud-bigdata-v2/_deconst.json). We should probably fix that there, instead?